examples/data/postproc_test/t070_mf2005/t070_mf2005.bas

```diff
@@ -0,0 +1,43 @@
+# BAS6 package for  MODFLOW-2005, generated by Flopy.
```

Comment:
You've included input and output files to be versioned within the flopy repo.  We try to avoid this if at all possible.  If files are to be versioned, then they should only be versioned if necessary.  For example, there should be no need to version this list files, the binary grid file, etc.  Or is there?

Reply:
I need most of them for the new tests I set up, but I agree that not all of them are needed (in particular, the .lst and .grb are indeed not needed).

If you think it is better, I could also just generate the model input and output files as part of the tests, and thus simply use the autotest/temp directory for that. I thought I would rather not do this in order to a have more focused testing, but I am happy to do this if you think it is better. If so, I could remove all the input and output files that I added to the repository.

Reply:
@etiennebresciani, I do think it would be better to generate these input and output files instead of storing them in the flopy repository.  Would you mind making that change?